### PR TITLE
Fixing the linter, first commit of 1.26.25.13

### PR DESCRIPTION
**Pull request by schuylermartin45**

### CODE REVIEW

**Review by ViridianMelody — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,13 +11,14 @@ source:
 
 build:
   noarch: python
```
Comment (ELundby45) [inline recipe/meta.yaml]:
```suggestion
  skip: True # [py<37]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,13 +11,14 @@ source:
...
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
     - python >=3.6
```
Comment (ELundby45) [inline recipe/meta.yaml]:
```suggestion
    - python
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,13 +11,14 @@ source:
...
     - pip
+    - wheel
   run:
     - python >=3.6
```
Comment (ELundby45) [inline recipe/meta.yaml]:
```suggestion
    - python
```

**Comment by schuylermartin45** [comment]
@ELundby45 Thanks for those, I'll double check the other `types` PR for those issues

**Review by ELundby45 — APPROVED** [review]

**Comment by schuylermartin45** [comment]
I accidentally duplicated this work: https://github.com/AnacondaRecipes/types-urllib3-feedstock/pull/2
Closing for favor of the earlier PR.

**Pull request closed without merging**